docs/src/main/sphinx/connector/jmx.rst

```diff
@@ -41,8 +41,7 @@ Commas in MBean names should be escaped in the following manner:
 .. code-block:: text
 
     connector.name=jmx
-    jmx.dump-tables=trino.memory:name=general\\,type=memorypool,\
-       trino.memory:name=reserved\\,type=memorypool
+    jmx.dump-tables=trino.memory:name=general\\,type=memorypool\\,trino.memory:name=reserved\\,type=memorypool
```

Comment:
The commas need to only be escaped if they are part of the table name IIRC. The logic seems to be defined [here](https://github.com/trinodb/trino/blob/master/plugin/trino-jmx/src/main/java/io/trino/plugin/jmx/JmxConnectorConfig.java#L44-L54).

```suggestion
    jmx.dump-tables=trino.memory:name=general\\,type=memorypool,trino.memory:name=reserved\\,type=memorypool
```

Reply:
Also this example is already correct. The `,\` at the end of line is actually escaping the value to allow splitting the value across a newline.